### ISSUE TITLE
Plan

### ISSUE THREAD

**Issue by robertdown** [openemr/OpenEMR-Documentation-Project#1]
Hi @gutiersa - what's the plan for this repo? We have a wiki system in place at https://open-emr.org/wiki and lots of documentation throughout the source code.

**Comment by gutiersa** [comment]
@robertdown 
Hi, 
ha ha, I am actually just playing with git. I am trying to learn it.
I did not even mean to save this under openemr. 

My idea is to organize the documentation, that is all.
Sometimes I have a hard time finding things (thank goodness for "search")
Also, I have noticed that at times some information is outdated in certain places. So I figured we could have a centralized location of documentation that would periodically feed the wiki, and the Documentation directory of the actual openemr package.
Here, those of us who are not programmers or developers can contribute. This repository can then populate the wiki, wherever that is located.
What do you think?

This is the file order I was thinking about:

```
FAQ
Awards
Demos
Compliance  per  region
	USA
		US Department  of  Health  and  Human Services
		ONC  Ambulatory  EHR  Certification 
		2011  ONC  Ambulatory  EHR  Certification 
			Complete  Certification 
		2014  ONC  Ambulatory  EHR  Certification 
			Complete  Certification 
			Modular  Certification 
		2015  ONC  Ambulatory  EHR  Certification
User Manual
	OpenEMR-5.0.1
		Initial  setup
		EMR
		Billing
		Patient  Accounts
		Office  Management
	OpenEMR-5.0.0
		Initial setup
		EMR
		Billing
		Patient  Accounts
		Office  Management
	OpenEMR-4.2.2
		Initial  setup
		EMR
		Billing
		Patient  Accounts
		Office  Management
	OpenEMR-4.2.1
	OpenEMR-4.2.0
	OpenEMR-4.1.2
	OpenEMR-4.1.1
	OpenEMR-4.1.0
	OpenEMR-4.0.0
	OpenEMR-3.2.0
	OpenEMR-3.1.0
System  Admin  Manual
	Support
	Stable  Production  Release
		Downloads
	Previous  Versions
	Compatibility
		Stack
		Server  OS
		PHP  Versions
		Web  Servers
		Databases
		Clients
		Browsers		
	Dependencies-Requirements
	Installation
		General
		Multi-site
	Securing  OpenEMR
	Configuring  OpenEMR
		Setup
			Databases
		Modules
		Patient  Portal
	Troubleshooting
		Installation
		Running
		Upgrading  and  Patching
Developer  Manual
OpenEMR  Architecture
Projects
Contributing
```

What do you think?

Sandra

**Comment by robertdown** [comment]
Hi Sandra,

We are actually in the midst of migrating our documentation - see this thread on the forums https://community.open-emr.org/t/documentation-migration-strategy/10097

**Comment by robertdown** [comment]
@gutiersa if you'd like, I'll bring you into the beta of the new site

**Comment by gutiersa** [comment]
@robertdown 
Yes, I would like to get involved. If there is one thing I admire about FreeBSD is that they have a documentation project. FreeBSD is one of the oldest open source projects. I think that ease of navigation through the documentation is important. I would like this for openemr. I sometimes have a hard time finding how to do things. So count me in.
thanks
Check out how this site is organized:
https://www.freebsd.org/

**Comment by gutiersa** [comment]
@robertdown
My idea above is different from the wiki though. It is kind of like versioning the "Documentation". Who knows, maybe it can be used to make hard copies of the user manual which could be sold by the non for profit.

**Comment by ghost** [comment]
@robertdown While you're moving over the wiki I'm going to write a couple of articles here / park them here since it's easier to write here than on MediaWiki. Most of these articles are stuff that I 2/3rds finished on the current MediaWiki. This will prep them for move over to Drupal.

Who knows maybe some of them can be blog posts too if you need the content.

**Comment by gutiersa** [comment]
Thanks for the update. I will look into it this coming week.
*Sandragutierrezmd*
